src/app/modules/profile/components/profile/profile.component.ts

```diff
@@ -1,15 +1,31 @@
 import { Component, OnInit } from '@angular/core';
+import { getUserRole } from 'src/app/redux/selectors/user.selectors';
+import { Store } from '@ngrx/store';
+import { State } from 'src/app/state/app.state';
 
 @Component({
   selector: 'app-profile',
   templateUrl: './profile.component.html',
-  styleUrls: ['./profile.component.scss']
+  styleUrls: ['./profile.component.scss'],
 })
 export class ProfileComponent implements OnInit {
+  getUserRole$ = this.store.select(getUserRole);
 
-  constructor() { }
+  isHR(role: any) {
+    role === 'HR' ? true : false;
+  }
+  isAdmin(role: any) {
+    role === 'administrator' ? true : false;
+  }
+  isCouch(role: any) {
+    role === 'couch' ? true : false;
```

Comment:
You can simply write `return role === 'HR';`

Reply:
Yes, you are right)